apps/setting/models_provider/impl/aws_bedrock_model_provider/credential/embedding.py

```diff
@@ -41,9 +26,6 @@ def is_valid(self, model_type: str, model_name, model_credential: Dict[str, obje
             return False
 
         try:
-            self._update_aws_credentials('aws-profile', model_credential['access_key_id'],
-                                         model_credential['secret_access_key'])
-            model_credential['credentials_profile_name'] = 'aws-profile'
             model: BedrockEmbeddingModel = provider.get_model(model_type, model_name, model_credential)
             aa = model.embed_query('你好')
             print(aa)
```

Comment:
No apparent irregularities, potential issues or optimizations were found in this code.